frontend/schemaCMS/src/modules/dataSource/dataSource.constants.js

```diff
@@ -6,12 +6,14 @@ export const FETCH_LIST_DELAY = 5 * 1000;
 export const DATA_SOURCE_FORM = 'data_source_form';
 export const DATA_SOURCE_NAME = 'name';
 export const DATA_SOURCE_FILE = 'file';
+export const DATA_SOURCE_LINK = 'link';
```

Comment:
if you're using this in a form, this name 'link' must be the same in the backend

Reply:
fixed